netkan/Dockerfile

```diff
@@ -8,12 +8,13 @@ RUN python -m unittest -v
 WORKDIR /
 USER root
 RUN rm -Rf /netkan
+ENTRYPOINT [".local/bin/netkan"]
+CMD ["--help"]
 
```

Comment:
Yep, this definitely belongs here. This will pave the way for some build improvements as this layout works, but isn't entirely optimal.